quicklink.module

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * @file
+ * Contains quicklink.module.
+ */
+
+use Drupal\Core\Routing\RouteMatchInterface;
+
+/**
+ * Implements hook_help().
+ */
+function quicklink_help($route_name, RouteMatchInterface $route_match) {
+  switch ($route_name) {
+    // Main module help for the quicklink module.
+    case 'help.page.quicklink':
+      $output = '';
+      $output .= '<h3>' . t('About') . '</h3>';
+      $output .= '<p>' . t('My Awesome Module') . '</p>';
+      return $output;
+
+    default:
+  }
+}
+
+function quicklink_preprocess_html(&$variables, $hook) {
+  if (!$variables["logged_in"]) {
```

Comment:
Hum, this relies on the user module defining this variable. There should be a more robust way to check what you want. You want this to be added to any page for anonymous users only?

Reply:
Correct, the JS prefetches each URL in the viewport. I don't want it prefetching every single admin page if the user has admin_menu installed. Reason being is that most of those pages will be uncached and could quickly bring an underpowered server to its knees. Thoughts?